orchestrator-bundle/orc8r-accessd-operator/tox.ini

```
@@ -0,0 +1,77 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+[tox]
+skipsdist=True
+skip_missing_interpreters = True
+envlist = lint, static, unit
+
+[vars]
+src_path = {toxinidir}/src/
+tst_path = {toxinidir}/tests/
+all_path = {[vars]src_path} {[vars]tst_path}
+
+[testenv]
+basepython = python3
+setenv =
+  PYTHONPATH = {toxinidir}:{toxinidir}/lib:{[vars]src_path}
+  PYTHONBREAKPOINT=ipdb.set_trace
+passenv =
+  PYTHONPATH
+  HOME
+  PATH
+  CHARM_BUILD_DIR
+  MODEL_SETTINGS
+  HTTP_PROXY
+  HTTPS_PROXY
+  NO_PROXY
+
+[testenv:fmt]
+description = Apply coding style standards to code
+deps =
+    black
+    isort
+commands =
+    isort {[vars]all_path}
+    black {[vars]all_path}
+
+[testenv:lint]
+description = Check code against coding style standards
+deps =
+    black
+    flake8
+    flake8-docstrings
+    flake8-copyright
+    flake8-builtins
+    pyproject-flake8
+    pep8-naming
+    isort
+commands =
+    pflake8 {[vars]all_path}
+    isort --check-only --diff {[vars]all_path}
+    black --check --diff {[vars]all_path}
+
+[testenv:static]
+description = Run static analysis checks
+deps =
+    -r{toxinidir}/requirements.txt
+    mypy
+    types-PyYAML
+    pytest
+    pytest-operator
+    juju
+    types-setuptools
+    types-toml
+commands =
+    mypy {[vars]all_path} {posargs}
+
+[testenv:unit]
+description = Run unit tests
+deps =
+    pytest
+    coverage[toml]
+    -r{toxinidir}/requirements.txt
+commands =
+    coverage run \
+      --source={[vars]src_path} \
+      -m pytest -v --tb native {posargs} {[vars]tst_path}
```

```suggestion
    coverage run --source={[vars]src_path} -m pytest -v --tb native -s {posargs}
```

By leaving `posargs` at the end, you can do things like `tox -e unit tests/test_charm.py::TestClass::test_case`

Once you have integration tests in place you just need to add `--ignore={[vars]tst_path}integration` to stop those being picked up :)

---

@Gmerold FYI, since I removed --ignore following a comment you had.